common/djangoapps/student/models.py

```diff
@@ -233,7 +233,10 @@ class UserProfile(models.Model):
 
     class Meta(object):
         db_table = "auth_userprofile"
-        permissions = (("can_deactivate_users", "Can deactivate, but NOT delete users"),)
+        permissions = (
+            ("can_deactivate_users", "Can deactivate, but NOT delete users"),
+            ("can_call_check_course_access_api", "Can access CheckCourseAccessView"),
```

Comment:
Where do the embargo models live? They don't live in this app. So why are the permissions in this app?